charts/v1.0/cray-nls/Chart.yaml

```diff
@@ -24,7 +24,7 @@
 
 apiVersion: v2
 name: "cray-nls"
-version: 1.4.48
+version: 1.4.49
```

Comment:
seems like we should stop doing this here so builds from this repo don't overwrite builds from the cray-nls-charts, right?